### PR TITLE
buddy:  scp Is Not Parsing user@node Properly

### PR DESCRIPTION
**Pull request by awly**
Buddy PR version of #6690

### CODE REVIEW

**Comment by awly** [comment]
@r0mant PTAL

File: tool/tsh/tsh.go [hunk 1]
```diff
@@ -1590,6 +1590,18 @@ func makeClient(cf *CLIConf, useProfileLogin bool) (*client.TeleportClient, erro
 				return nil, err
 			}
 		}
+	} else if cf.CopySpec != nil {
```
Comment (r0mant) [inline tool/tsh/tsh.go:1593]: I think `len(cf.CopySpec) != 0` would be a more reliable check.

File: tool/tsh/tsh.go [hunk 2]
```diff
@@ -1590,6 +1590,18 @@ func makeClient(cf *CLIConf, useProfileLogin bool) (*client.TeleportClient, erro
 				return nil, err
```
Comment (r0mant) [inline tool/tsh/tsh.go:1590]: `trace.Wrap(err)`

**Review by r0mant — APPROVED** [review]

**Review by russjones — APPROVED** [review]
Bot.

**Pull request merged**